### DISCUSSION TITLE
tutorials/voyage-part3/10-observability-with-apollo-studio

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#76] [course feedback]
# tutorials/voyage-part3/10-observability-with-apollo-studio

https://www.apollographql.com/tutorials/voyage-part3/10-observability-with-apollo-studio

**Comment by konstantinkrumin** [comment]
I would like to say how amazing this whole course series is! I'm a fullstack developer and previously I had very limited experience with Apollo GraphQL, however, the current new job I started is using Apollo GraphQL extensively. This motivated me to look for a Apollo GraphQL tutorial and I'm very happy that I came across yours.

Truly amazing job done in delivering the content. The whole series for both Associate and Developer certification programs provide a perfect combination of interactivity and content delivery. I love the fact that videos, text, assignments and coding exercises all provided as part of this program. And finally, getting certificates of completion that I can add to my LinkedIn is a nice cherry of top!

I would like to say big thanks to all people who have worked on this course and I can 100% say that is an exemplary program. It definitely helped me to go from almost 0 knowledge of GraphQL to become a quite confident user.

**Reply by mabuyo** [reply]
Thank you so much for the wonderful review! I'm so glad you learned a lot from the courses 😊

**Reply by sushant-rathor** [reply]
I totally agree with Konstantin and share and echo similar sentiments and appreciation for this course. Thanks  Michelle/team for this wonderful learning experience. Cheers

**Comment by ranazaincode** [comment]
when will complete the lesson, how get the certfication.

**Reply by mabuyo** [reply]
Hi there @RanaZain1998 - take a look at your [dashboard for certifications](https://www.apollographql.com/tutorials/dashboard/certifications). It should show you which courses need to be completed!
